feature/search/src/main/kotlin/uk/govuk/app/search/data/remote/model/Result.kt

```diff
@@ -4,6 +4,6 @@ import com.google.gson.annotations.SerializedName
 
 data class Result(
     @SerializedName("title") val title: String,
-    @SerializedName("description") val description: String,
+    @SerializedName("description_with_highlighting") val description: String?,
```

Comment:
Is this going to cause an issue if we have to switch back to v1?

Reply:
Yup, `description` simply doesn't exist. I could make this `DESCRIPTION` and map it in the SearchConfig - it'll help a bit - but fundamentally if we're switching back it'll not be as quick and easy as it should have been

Reply:
I've added another commit to do as suggested above